.mocharc.json

```diff
@@ -1,5 +1,5 @@
 {
-  "spec": "packages/*/!(integration-tests)/test/*.js",
+  "spec": "packages/*/!(integration-tests)/test/{*.js,**/*.{test,spec}.js}",
```

Comment:
There are failing unit-tests on this PR that fail because they were not running.

Reply:
I have fixed it here - https://github.com/parcel-bundler/parcel/pull/9658#:~:text=Fix%20test%20that%20was%20not%20running